webview/__init__.py

```diff
@@ -67,8 +68,10 @@
 
 token = _token
 windows = []
+menus = []
 
-def start(func=None, args=None, localization={}, gui=None, debug=False, http_server=False, user_agent=None):
+def start(func=None, args=None, localization={}, gui=None, debug=False, http_server=False, user_agent=None,
+          app_menu_list=[]):
```

Comment:
`app_menu_list=` could be just `menu=` for clarity. Also would it be possible to make a menu window not app specific? Ideally both `create_window` and `start` should accept `menu` param. The `create_window` menu would a be window specific. If both are specified, the `create_window` menu would replace the global menu.

Reply:
I spent some time trying to get window-specific menu working on all platforms and I haven't been able to get it to work fully, so I'm going to leave that out of this PR.